src/app/shared/components/ShareBar/index.tsx

```diff
@@ -51,19 +52,26 @@ const ShareBar = (props: ShareBarProps) => {
 
   const router = useRouter()
 
+  // WIP
+  // Can't use just editId & userId here as need to ensure its the current users team sheet to allow them to edit it!
```

Comment:
i thought we could use our existing findOrCreateUser here but its not so easy because this is a client component

we can discuss today